tests/cypress/integration/page-ordering.test.js

```diff
@@ -0,0 +1,45 @@
+describe("Test Page Order Change", () => {
+	it("Can change parent pages order", () => {
+		cy.login();
+		cy.visit("/wp-admin/edit.php?post_type=page");
+
+		const first = '.wp-list-table tbody tr:nth-child(1)';
+		const second = '.wp-list-table tbody tr:nth-child(2)';
+		let firstText = cy.get(`${first} .row-title`);
+		let secondText = cy.get(`${second} .row-title`);
+
+		cy.get(first).drag(second);
+		// wait for order update done.
+		cy.get(`${second}  .check-column input`).should('exist');
+
+		cy.get(`${first} .row-title`).then($el => {
+			secondText.should('have.text', $el.text());
+		});
+		cy.get(`${second} .row-title`).then($el => {
+			firstText.should('have.text', $el.text());
+		});
+
+	});
+
+	it("Can change Child pages order", () => {
+		cy.login();
+		cy.visit("/wp-admin/edit.php?post_type=page");
+	
+		const first = '.wp-list-table .level-0 + .level-1';
+		const second = '.wp-list-table .level-0 + .level-1 + .level-1';
+		const firstText = cy.get(`${first} .row-title`).invoke('text');
+		const secondText = cy.get(`${second} .row-title`).invoke('text');
+
+		cy.get(first).drag(second);
+		// wait for order update done.
+		cy.get(`${second}  .check-column input`).should('exist');
+		
+		cy.get(`${first} .row-title`).then($el => {
+			secondText.should('have.text', $el.text());
+		});
+		cy.get(`${second} .row-title`).then($el => {
+			firstText.should('have.text', $el.text());
+		});
+	  });
+});
+  
```

Comment:
The extra blank line can be removed at the EOF.

Reply:
As per rule here https://github.com/10up/simple-page-ordering/blob/develop/.editorconfig#L9, new line will be always there at the EOF